### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yy2so**
New read me

Fixes #ISSUSE_ID.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # [ShardingSphere - Building a Criterion and Ecosystem Above Multi-Model Databases](https://shardingsphere.apache.org/)
 
-**Official Website:** [https://shardingsphere.apache.org/](https://shardingsphere.apache.org/)
+**Official Website:** [https://shardingsphere.apache.org/]()
```
Comment (tristaZero) [inline README.md:3]: This is a markdown grammar. Your change will trigger a bad redirection.

File: README.md [hunk 2]
```diff
@@ -9,13 +9,9 @@
...
-[![Stargazers Over Time](https://starchart.cc/apache/shardingsphere.svg)](https://starchart.cc/apache/shardingsphere)
-
-**Contributors Over Time**
-
-[![Contributors Over Time](https://contributor-graph-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=apache/shardingsphere)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=apache/shardingsphere)
+**Stargazers Over Time**           |  **Contributors Over Time**
+:---------------------------------:|:------------------------------------:
+[![Stargazers over time](https://starchart.cc/apache/shardingsphere.svg)](https://starchart.cc/apache/shardingsphere)  |  [![Contributor over time](https://contributor-graph-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=apache/shardingsphere)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=apache/shardingsphere)
 
 [![Build Status](https://api.travis-ci.org/apache/shardingsphere.svg?branch=master&status=created)](https://travis-ci.org/apache/shardingsphere)
```
Comment (tristaZero) [inline README.md:16]: Why don't these badges show up together? e.g., `Gitter` and `buld status`.

File: README.md [hunk 3]
```diff
@@ -80,39 +76,28 @@ Apache ShardingSphere includes 3 independent products: JDBC, Proxy & Sidecar (Pl
...
-Applicable in any ORM framework based on JDBC, such as JPA, Hibernate, Mybatis, Spring JDBC Template or direct use of JDBC. Supports any third-party database connection pool, such as DBCP, C3P0, BoneCP, Druid, HikariCP. Supports any kind of JDBC standard database: MySQL, Oracle, SQLServer, PostgreSQL and any SQL92 followed databases.
+A lightweight Java framework providing extra services at the Java JDBC layer. With the client end connecting directly to the database, it provides services in the form of a jar and requires no extra deployment and dependence. Supports any third-party database connection pool, such as DBCP, C3P0, BoneCP, Druid, HikariCP. Supports any kind of JDBC standard database: MySQL, Oracle, SQLServer, PostgreSQL and any SQL92 followed databases.
 
-![](./docs/document/content/Read-Me-Pics/Read-me-JDBC.png)
+For more details, follow this [link to the official website](https://shardingsphere.apache.org/document/current/en/overview/).
```
Comment (tristaZero) [inline README.md:81]: `https://shardingsphere.apache.org/document/current/en/overview/#shardingsphere-jdbc` may bring users to an exact position about JDBC, same with proxy.

File: README.md [hunk 4]
```diff
@@ -80,39 +76,28 @@ Apache ShardingSphere includes 3 independent products: JDBC, Proxy & Sidecar (Pl
...
 
-ShardingSphere-JDBC adopts a decentralized architecture, applicable to high-performance light-weight OLTP application developed with Java. ShardingSphere-Proxy provides static entry and all languages support, suitable for OLAP application and sharding databases management and operation.
+ShardingSphere-JDBC adopts a decentralized architecture, applicable to high-performance light-weight OLTP applications developed with Java. ShardingSphere-Proxy provides static entry and all languages support, suitable for OLAP application and sharding databases management and operation.
 
 Through the mixed use of ShardingSphere-JDBC & ShardingSphere-Proxy together with a unified sharding strategy by the same registry center, the ShardingSphere ecosystem can build an application system suitable to all kinds of scenarios.
 
-![](./docs/document/content/Read-Me-Pics/Read-me-hybrid.png)
+More details can be found following this [link to the official website](https://shardingsphere.apache.org/document/current/en/overview/).
 
 ## Roadmap:
```
Comment (tristaZero) [inline README.md:102]: It is suggested to place `Solution` above `Roadmap` to connect closely to the products introduced above.

**Review by tristaZero — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yy2so) [reply README.md]: Noted, fixed

File: README.md (hunk 2 above)
Comment (yy2so) [reply README.md]: Changed

File: README.md (hunk 3 above)
Comment (yy2so) [reply README.md]: Good tip, fixed

File: README.md (hunk 4 above)
Comment (yy2so) [reply README.md]: Changed

**Review by tristaZero — APPROVED** [review]

**Pull request merged**